frontend/src/scenes/insights/FunnelCorrelation.tsx

```diff
@@ -34,6 +37,16 @@ export const FunnelCorrelation = (): JSX.Element | null => {
         return null
     }
 
+    if (!correlationAnalysisAvailable) {
+        return (
+            <PayCard
+                identifier={AvailableFeature.CORRELATION_ANALYSIS}
+                title="Get a deeper understanding of what your users are doing"
+                caption="Correlation Analysis allows you to find signals for conversions and dropoffs that help you gain deeper insights."
```

Comment:
```suggestion
                caption="Correlation Analysis automates analysis over your events to find signals for why users are converting or dropping off."
```

Reply:
Wanted to try and make it clearer - "deeper insights" feels a little vague

Reply:
Yes, I like this. (I spent literally 3 seconds thinking about the copy)